### DISCUSSION TITLE
Changelogs won't show when logged out

### DISCUSSION THREAD

**Discussion by TheRealStevie** [ppy/osu#15415] [Q&A] [answered]
Changelogs won't appear while you're offline. It used to show them before in the past updates.

Offline;
![osu_2021-11-03_01-51-41](https://user-images.githubusercontent.com/75444413/139911222-10b867dd-adb2-4a5e-9793-c7a7067e5625.png)

Online:
![osu_2021-11-03_01-52-55](https://user-images.githubusercontent.com/75444413/139911295-9d2202ee-385a-4f96-9874-13d2ff654a31.png)

Logs:
[database.log](https://github.com/ppy/osu/files/7462658/database.log)
[network.log](https://github.com/ppy/osu/files/7462660/network.log)
[performance.log](https://github.com/ppy/osu/files/7462662/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7462665/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7462668/updater.log)

**Comment by peppy** [comment, marked as answer]
Will be fixed via #15431.

**Reply by TheRealStevie** [reply]
I can confirm on the latest update that changelogs appear once again. Thanks!
